search_assistant/src/search_assistant.py

```diff
@@ -147,26 +138,26 @@ def init_llm_model(self) -> None:
         Initializes the LLM endpoint
 
         Returns:
-        llm (SambaStudio or Sambaverse): Langchain LLM to use
+        llm (SambaStudio or SambaNovaCloud): Langchain LLM to use
         """
-        fastapi_url = DEFAULT_FASTAPI_URL
         if self.prod_mode:
-            fastapi_api_key = st.session_state.FASTAPI_API_KEY
+            sambanova_api_key = st.session_state.SAMBANOVA_API_KEY
```

Comment:
Same comment as for EKR

Reply:
this is also present in financial assistant and function calling, Should we handle this in a following PR?